docs-src/content/sessions/update.md

```diff
@@ -236,7 +252,15 @@ client.setStory(token: token, storyID: storyID, params: params) { (sessionOutput
 {{< /tab >}}
 
 {{< tab "Java/Android" "java" >}}
-// Example coming soon!
+SetStory story = SetStory.newBuilder()
```

Comment:
Define the local variables (`storyId`, `params`) to make it clearer what types they are.